packages/notifi-dapp-example/src/components/ConfigAlertModal.tsx

```diff
@@ -4,26 +4,31 @@ import {
 } from '@notifi-network/notifi-react-card';
 import React from 'react';
 
+import { AlertSubscription } from './AlertSubscription';
+
 export const ConfigAlertModal = () => {
   const { updateFtuStage } = useNotifiSubscriptionContext();
 
   return (
     <div className="h-4/6 w-4/6 bg-notifi-container-bg rounded-2xl flex flex-col items-center justify-between mb-8 shadow-container">
       <div>
         <div className="flex flex-col items-center justify-center">
-          <p className="font-bold text-xs opacity-50 mt-2.5">Step 3 of 3</p>
-          <p className="font-bold text-2xl mt-6">
-            Select alerts you want to receive
-          </p>
+          <p className="font-semibold text-xs opacity-50 mt-2.5">Step 3 of 3</p>
+          <p className="text-2xl mt-6">How do you want to be notified?</p>
```

Comment:
I feel like the subtitle may be incorrect(in figma), better to check with Sam later